src/cloud/howtos/debug.md

```diff
@@ -317,13 +317,23 @@ To debug CLI commands:
    export PHP_IDE_CONFIG="serverName=<name of the server that is configured in PHPSTORM>"
    ```
 
-These variables are removed when the SSH session ends. When adding the variables, you can add runtime options:
+   These variables are removed when the SSH session ends.
+   If you expect to use SSH and debug multiple times, you can put the export commands into a bash script in the `/tmp` directory to run them each time.
```

Comment:
I think that it’s enough to indicate that the variables are related to the session.
I propose to remove this: 
"If you expect to use SSH and debug multiple times, you can put the export commands into a bash script in the `/tmp` directory to run them each time."

Reply:
removed